Problems/Problems/ReverseLinkedList.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace Problems
+{
+    
+    // Definition for singly-linked list.
+
+    public class Node { 
+        public int val; 
+        public Node next;
+        public Node(int val=0, Node next=null) {
+            this.val = val;
+            this.next = next;
+        }
+}
+
+    class Program
+    {
+        private Node Head = null;
+
+        public void AddNode(Node head, Node node)
+        {
+            if (head == null)
+            {
+                this.Head = node;
+                return;
+            }
+            if (head.next == null)
+            {
+                head.next = node;
+                return;
+            }
+            this.AddNode(head.next, node);
+        }
+        
+        public void PrintList(Node head)
+        {
+            Console.Write("[");
+            while (head.next != null)
+            {
+                Console.Write(head.val + ",");
+                head = head.next;
+            }
+            Console.Write(head.val + "]");
+            
+        }
+        
+        //recursive
+        public void ReverseList(Node prev, Node head)
+        {
+            if (head.next == null)
+            {
+                this.Head = head;
+                head.next = prev;
+                return;
+            }
+            ReverseList(head, head.next);
+            head.next = prev;
+        }
+        
+
+        static void Main(string[] args)
+        {
+            Program List = new Program();
```

Comment:
Naming convention: list [camelCase]
Check this out:
https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md